roles/matrix-server/defaults/main.yml

```diff
@@ -203,6 +210,10 @@ matrix_riot_web_enabled: true
 matrix_riot_web_default_identity_server_url: "https://{{ matrix_synapse_trusted_third_party_id_servers[0] }}"
 
 
+# Matrix mautrix is a Matrix <-> Telegram bridge
+matrix_mautrix_telegram_enabled: false
```

Comment:
Maybe we should add some default values for `matrix_mautrix_telegram_api_id` and `matrix_mautrix_telegram_api_hash` here (like the ones you have in `examples/host-vars.yml` now).

These defaults won't be useful unless changed, but it's a good way to document all variables that this integration requires in a single place, making it discoverable (in addition to being discoverable by reading the docs file).

This way, if someone runs the playbook without redefining the variables, at least it won't explode with errors.

Right now, this would cause trouble for existing users of the playbook, unless they merge `examples/host-vars.yml` with what they already have. Most users should not need to care about defining any ``matrix_mautrix_telegram_*` variables, unless they want to use that integration.